templates/cluster-override.conf

```diff
@@ -3,12 +3,16 @@
 db {
   backend = "rlog"
   role = "core"
+  tlog_push_mode = "{{ push_mode | default('async') }}"
+  rpc_module = "{{ rpc_module | default('gen_rpc') }}"
 }
 {% elif emqx_db_role == "replicant" %}
 db {
   backend = "rlog"
   role = "replicant"
-  core_nodes = "{{ cores }}"
+  core_nodes = "emqx@{{ cores | join(', emqx@' )}}"
```

Comment:
It's a horrible hack

Reply:
Maybe we can define another var in a similar way to what is done for `cores`?

https://github.com/k32/emqx-ansible-cdk-helpers/blob/409d73acceaddca8e78d7a56b026655909534706/tuning.yml#L80

Reply:
Actually: is this change needed? It has been working using only `cores` so far. :thinking: